vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -1,165 +1,56 @@
 
-The Entando CLI (ent) automates many of the tasks involved in deploying an Entando Bundle, but you can also choose to perform these tasks manually.
+1. Building a Docker image from a microservice
+2. Defining the bundle component
+3. Pushing the bundle artifacts to a Docker registry
+4. Applying the bundle custom resource to Kubernetes
+5. Installing the bundle into an Entando Application
 
-## Prerequisites
-* Use the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment) to verify you have the prerequisites in place (e.g. Java, npm, git)
-``` sh
-ent check-env develop
-```
-* Your Git credentials
-* A Git repository
-* An Entando instance
-* An Entando project directory, either [developed by hand](./publish-simple-bundle.md) or [generated by the Entando Component Generator](../ms/generate-microservices-and-micro-frontends.md)
-
-## CLI Steps
-The following steps make use of the Entando `ent prj` command and its publication system (pbs) convenience methods. See the [Manual Steps](#manual-steps) section below for a more detailed description of the underlying tasks.
-
-1. Build the project using the `ent prj` command. This saves you from having to build each part of the project individually. If you are using a project directory with just a bundle child directory (e.g. a bundle with hand-built components or exported from another environment), then there is nothing to build and you should skip this step.
-``` sh
-ent prj build
-```
-::: tip
-The first run can take longer due to node downloads for any MFE widgets. You can use `ent prj fe-build` or `ent prj be-build` for subsequent runs to independently build just the frontend or backend components.
-
-:::
-
-2. Initialize the bundle directory
-``` sh
-ent prj pbs-init
-```
-
-3. Publish the build artifacts to GitHub and Docker Hub  
-``` sh
-ent prj pbs-publish
-```
-
-4. Deploy the bundle into the ECR
-``` sh
-ent prj deploy
-```
-Jump to [Install the Bundle into an Application](#install-the-bundle-into-an-application) below to finish installing your bundle.
-
-## Manual Steps
-
-### Build a Docker Image for Microservices
-1. Build the project from the project directory
- ```sh
- ./mvnw -Pprod clean package jib:dockerBuild
-````
-
-> **Note**
->
-> By default, the organization used to generate the Docker image is `entando`. You can customize this value during project initialization by either changing the `pom.xml` file or providing the `-Djib.to.image=<org>/<name>:<version>` to the `jib:dockerBuild` command.
-
-> **Note**
->
-> The output image name is generated using the organization value defined during project initialization. You can override the provided values by altering the `pom.xml` file or by customizing the `-Djib.to.image` parameter used by the `./mvnw` command.
+The Entando Bundle CLI tool (**ent bundle**) automates many of the tasks involved in deploying an Entando project bundle.
```

Comment:
@nshaw, should there be a mention and link to how to do this for V1  bundles?

Reply:
Just to document, we discussed on the call and decided it's not needed here.